core/src/main/java/org/apache/gravitino/catalog/EntityCombinedTable.java

```diff
@@ -128,6 +128,14 @@ public boolean imported() {
     return imported;
   }
 
+  public Table tableFromCatalog() {
```

Comment:
How about `tableFromExternalCatalog`?

Reply:
I would keep the current name, because it introduces the concept of "External", if finally we have external and managed catalog in Gravitino, then the method name here will be confusing.